docs/src/modules/java/pages/actions-as-controller.adoc

```diff
@@ -0,0 +1,52 @@
+= Actions as Controller
+
+Actions can be used to implement link:https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller[MVC Controllers] by
+acting as the external interface of a service, receiving requests, operating over the requests values and forwarding the call
+to other components.
+
+== Defining the `proto` file
+
+To illustrate how you can use an Action as a Controller, we will build on top of the xref:value-entity.adoc[Value Entity Counter] example and add
+an Action that will take the incoming `Increase` command, double its value and forward to the request to `Counter` entity.
+
+Note that in this example we are adding an Action to an existing service, the Value Entity Counter service.
```

Comment:
that example is used in value-entity.adoc so would be good with a link

Reply:
I linked in the beginning of page. Will link again.